extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -331,7 +331,7 @@ export class AmpA4A extends AMP.BaseElement {
      */
     this.fromResumeCallback = false;
 
-    /** @protected {string} */
+    /** @type {string} */
```

Comment:
Why change the visibility on this?

Reply:
I was getting errors with it as protected